modules/slugs/includes/qtranslate-slug-settings-options.php

```diff
@@ -35,7 +35,7 @@ function get_multi_txt_choices() {
  * @return array
  */
 function qts_options_page_fields() {
-    $post_types = get_post_types( array( '_builtin' => false, 'public' => true ), 'objects' );
+    $post_types = get_post_types( array( 'public' => true ), 'objects' );
```

Comment:
What is the effect of adding the builtin post types? Why were they filtered out before?

Reply:
issue with some `_builtin` type (e.g. post and pages) is that slugs cannot be modified (hence customized slug will apparently just have no effect), but correct indication is not the `_builtin` type itself (which may include also types with slugs that could be modified), but the `rewrite=false` which is actually used to filter the post/taxonomy types in `qts_options_page_build_slug_fields` later.